tests/cucumber/steps/steps.js

```diff
@@ -5419,6 +5426,29 @@ module.exports = function getSteps(options) {
     }
   );
 
+  Then(
+    'the current application should have the following boxes {string}.',
+    async function (boxNames) {
+      let boxes;
+      if (boxNames !== '') {
+        boxes = splitAndProcessAppArgs(boxNames);
```

Comment:
nit: consider refactoring this logic into the `splitAndProcessAppArgs` method.

Reply:
good point - done!